schemas/rule.yml

```
@@ -0,0 +1,84 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used to generate alerts or other notable events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts oor other notable events.
```

```suggestion
   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts ooor other notable events.
```

What about this? ;-)

---

Kidding, I prefer "or" :-)

---

DOH! fixed